### PR TITLE
feature/Day3

### PR DESCRIPTION
**Pull request by AbhishekPandey-successive**

### CODE REVIEW

**Review by Rohit0203 — COMMENTED** [review]
Why there are multiple instance of server listening. 
There should only one instance of sever listening. Fix all changes , before making any PR again

File: Day3/userGeolocation.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  });
+
+});
+
+
+
+
+
+
+app.listen(3600, () => console.log('Example app listening on port 3600'))
```
Comment (Rohit0203) [inline Day3/userGeolocation.js]: Multiple instance of app.listen . Change it

**Comment by AbhishekPandey-successive** [comment]
> Why there are multiple instance of server listening. There should only one instance of sever listening. Fix all changes , before making any PR again

I have make changes by creating the .env file and defining the port and listening all at single instance.

**Review by Rohit0203 — APPROVED** [review]
Need more improvement in code readability

File: Day3/userGeolocation.js (hunk 1 above)
Comment (AbhishekPandey-successive) [reply Day3/userGeolocation.js]: Okay  sir I will focused on it.

**Pull request merged**